src/devices/lacrosse_th3.c

```diff
@@ -112,14 +134,14 @@ static int lacrosse_th3_decode(r_device *decoder, bitbuffer_t *bitbuffer)
 
     /* clang-format off */
     data = data_make(
-            "model",            "",                 DATA_STRING, "LaCrosse-TH3",
-            "id",               "Sensor ID",        DATA_FORMAT, "%06x", DATA_INT, id,
-            "seq",              "Sequence",         DATA_INT,     seq,
-            "flags",            "unknown",          DATA_INT,     flags,
-            "temperature_C",    "Temperature",      DATA_FORMAT, "%.1f C",  DATA_DOUBLE, temp_c,
-            "humidity",         "Humidity",         DATA_FORMAT, "%u %%", DATA_INT, humidity,
-            "mic",              "Integrity",        DATA_STRING, "CRC",
-            NULL);
+         "model",            "",                 DATA_FORMAT, "LaCrosse-TH%u", DATA_INT, model,
```

Comment:
This isn't great. The data extractor scripts look for fixed strings. It should be `model==3?"LaCrosse-TH3":"LaCrosse-TH2"`

Reply:
Let's resolve that then. What do you mean with 'data extractor scripts'.

Reply:
The python script that gathers the json data for https://triq.org/explorer/ -- I guess I need to clean up and commit that then ;)